src/util/secrets/local.c

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
I sincerely don't get why this whole file has been added at this point as it's not used at all in this patch.

Reply:
it was an artifact of my previous work that I forgot to git rm. Fixed.